http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoPubsubMessageSubscriber.scala

```diff
@@ -323,6 +362,23 @@ class LeoPubsubMessageSubscriber[F[_]: Async: Timer: ContextShift: Concurrent](
             _ <- dbRef.inTransaction(clusterQuery.setToStopping(msg.runtimeId, now))
             _ <- runtimeConfig.cloudService.interpreter
               .stopRuntime(StopRuntimeParams(RuntimeAndRuntimeConfig(runtime, runtimeConfig), now))
+             _ <- if (runtimeConfig.cloudService == CloudService.GCE) //TODO: we should cover dataproc as well in the future
+              // We're running this asynchronously because we don't want to hold up acking event since Stopping an instance can take a while
+               F.runAsync(gceRuntimeMonitor.process(msg.runtimeId).compile.drain) {
```

Comment:
Kinda cool we can express the whole stop -> update -> start transition here without generating separate pubsub messages.